app/views/accounts/_member_table.html.haml

```diff
@@ -29,5 +29,5 @@
 
         %td= Users::NamePresenter.new(au.user, username_label: true).full_name
         %td= au.user.email
-        %td= au.user_role
+        %td= t(".role.#{au.user_role.downcase.parameterize.underscore}")
```

Comment:
It's not ideal, but it's probably the most expedient rather that trying to fully refactor it into something better. Maybe an interim step would be to pull this up into an `AccountUser#user_role_key` that holds the `user_role.downcase.parameterize.underscore`. That could make future refactoring easier.